gridData/core.py

```diff
@@ -346,11 +346,29 @@ def export(self, filename, file_format=None):
             pickle (use :meth:``Grid.load` to restore); :meth:`Grid.save`
             is simpler than ``export(format='python')``.
 
+        Parameters
+        ----------
+        filename : str
+            name of the output file
+        file_format : {'dx', 'pickle', None} (optional)
+            output file format, the default is "dx"
+        type : str (optional)
+            for DX, set the output DX array type, e.g., "double" or
+            "float"; note that PyMOL only understands "double" (see
+            issue `#35`_). By default (``None``), the DX type is
+            determined from the numpy dtype of the array of the grid
+            (and this will typically result in "double").
+
+            .. versionadded:: 0.4.0
+
+
+        .. _`#35`: https://github.com/MDAnalysis/GridDataFormats/issues/35
+
         """
         exporter = self._get_exporter(filename, file_format=file_format)
-        exporter(filename)
+        exporter(filename, type=type)
 
-    def _export_python(self, filename):
+    def _export_python(self, filename, **kwargs):
```

Comment:
where is `kwargs` used in the function? Wouldn't explicit named args be better?

Reply:
See https://github.com/MDAnalysis/GridDataFormats/pull/36#discussion_r157713756

I could add `type=None` and then ignore it but it seems clearer (to me, at least...) that `**kwargs` means that the method does not know what it will get and it will just take what it needs.

Reply:
I added an explanatory comment in cd3fdb051b86d3a2559461411cbcec60f98102d7